CMakeLists.txt

```diff
@@ -34,22 +34,14 @@ option(MICM_ENABLE_COVERAGE "Enable code coverage output" OFF)
 option(MICM_ENABLE_MEMCHECK "Enable memory checking in tests" OFF)
 option(MICM_ENABLE_JSON "Enable json configuration file reading" ON)
 option(MICM_BUILD_DOCS "Build the documentation" OFF)
-option(MICM_ENABLE_CUDA "Build with Cuda support" OFF)
-option(MICM_ENABLE_OPENACC "Build with OpenACC Support" OFF)
 option(MICM_ENABLE_LLVM "Build with LLVM support for JIT-compiling" OFF)
 option(MICM_ENABLE_TESTS "Build the tests" ON)
 option(MICM_ENABLE_EXAMPLES "Build the examples" ON)
 option(MICM_ENABLE_PROFILE "Profile MICM Solver" OFF)
+set(MICM_GPU_TYPE "None" CACHE STRING "The GPU type being targeted")
```

Comment:
shall we lower case the `MICM_GPU_TYPE` here or somewhere later?

Reply:
Apologies!  Forgot to add that until just now in my latest commit.  I added it in the dependencies.cmake since that is where we are doing our checking but am happy to move it elsewhere.

Reply:
Why lowercase this option?

Reply:
so we can allow a user to provide either `a100` or `A100`